### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by dimitris-t**
"inline highligh" -> "inline highlight"

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Make a new "filename.md" within the category folder within `/docs/wiki/CATEGORY/
 ### Wiki style tips
 
 - Inline code highlighting (`$ echo 'this is inline'`) does not look the best in RTD, try to have as little inline syntax highlighting as possible.
-- **osqueryd**, **osqueryi** and other tool names should be in bold. Use `inline highligh` when a tool or script is mentioned for the first time.
+- **osqueryd**, **osqueryi** and other tool names should be in bold. Use `inline highlighτ` when a tool or script is mentioned for the first time.
```
Comment (muffins) [inline docs/README.md]: Not sure what kind of character set that is, but this is rendering as the greek Tao character to me. Mind changing this to be the en-us character set for `t`?
```
>>> 'highlighτ'
'highligh\xcf\x84'
```

**Comment by dimitris-t** [comment]
How embarrassing! I think I fixed it now.

**Comment by muffins** [comment]
ok to test

**Comment by osqueryer** [comment]
:-1: The commit 16cc47228e0fb159360c3f64b545036c8ba15c03 (Job results: [3677](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3677/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 16cc47228e0fb159360c3f64b545036c8ba15c03 (Job results: [3678](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3678/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:no_good: The commit 16cc47228e0fb159360c3f64b545036c8ba15c03 (Job results: [3294](https://jenkins.osquery.io/job/osqueryPullRequestAudit/3294/)) failed the code audit or documentation test.

**Comment by osqueryer** [comment]
:-1: The commit 16cc47228e0fb159360c3f64b545036c8ba15c03 (Job results: [6782](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6782/)) failed one or more tests (**Linux**).

**Comment by theopolis** [comment]
ok to test

**Comment by osqueryer** [comment]
:-1: The commit 16cc47228e0fb159360c3f64b545036c8ba15c03 (Job results: [3679](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3679/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 16cc47228e0fb159360c3f64b545036c8ba15c03 (Job results: [3680](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3680/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 16cc47228e0fb159360c3f64b545036c8ba15c03 (Job results: [6784](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6784/)) failed one or more tests (**Linux**).

**Pull request merged**